### DISCUSSION TITLE
Plans for multi-dimensional partitions (with more than two dimensions)

### DISCUSSION THREAD

**Discussion by clement-uspace** [dagster-io/dagster#20283] [Q&A] [answered]
Hi, 
I see that currently MultiPartitions only support the cross-product of two other partitions. Are there any plans to support an arbitrary number of dimensions ?

**Comment by alangenfeld** [comment, marked as answer]
No currently active plans that I am aware of. You can file an issue to request the feature (I can't find an existing issue).

**Reply by clement-uspace** [reply]
Okay thanks, I did that [here](https://github.com/dagster-io/dagster/issues/20356)
